src/scaffolding/master.scaffolding.hbs

```diff
@@ -57,6 +57,10 @@
     </div>
 </section>
 
+<section class="scroll-animation">
+	{{bioInclude 'components/BioScrollAnimation/scaffolding/BioScrollAnimation' }}
```

Comment:
IMO wrapping this in a bioInclude is unnecessary because the only content of the scaffolding is the web-component anyway.